overlord/hookstate/ctlcmd/install.go

```diff
@@ -0,0 +1,63 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package ctlcmd
+
+import (
+	"github.com/snapcore/snapd/i18n"
+)
+
+var (
+	shortInstallHelp = i18n.G("Install snaps and components")
+	longInstallHelp  = i18n.G(`
+The install command installs snaps and components.
+
+WARNING: currently this command is implemented only for components.
```

Comment:
Based on the conversation about removing `yet` from the error message when attempting to install a snap, should this text be reworded as well?

Reply:
Good point, changed now